frontend/amundsen_application/static/js/pages/TableDetailPage/ReportTableIssue/index.tsx

```diff
@@ -112,7 +118,13 @@ export class ReportTableIssue extends React.Component<
     this.setState({ isOpen: !this.state.isOpen });
   };
 
+  onPriorityChange = (event) => {
```

Comment:
Nit: I would call this 'handlePriorityChange', to differentiate from the prop function.